src/Mod/Sketcher/Gui/EditModeConstraintCoinManager.cpp

```diff
@@ -677,16 +677,13 @@ void EditModeConstraintCoinManager::processConstraints(const GeoListFacade & geo
                                         Base::Vector3d l2p1 = lineSeg->getStartPoint();
                                         Base::Vector3d l2p2 = lineSeg->getEndPoint();
                                         double radius = circleSeg->getRadius();
-                                        pnt2.ProjectToLine(ct-l2p1, l2p2-l2p1); //project on the line translated to origin
-                                        Base::Vector3d dir = pnt2;
+                                        pnt1.ProjectToLine(ct-l2p1, l2p2-l2p1); //project on the line translated to origin
+                                        Base::Vector3d dir = pnt1;
                                         dir.Normalize();
-                                        pnt1 = ct + dir * radius;
-                                        pnt2 += ct;
+                                        pnt2 = ct + dir * radius;
+                                        pnt1 += ct;
                                         Base::Vector3d d = l2p2 - l2p1;
                                         double ActDist = std::abs(-ct.x*d.y+ct.y*d.x+l2p1.x*l2p2.y-l2p2.x*l2p1.y) / d.Length() - radius;

```

Comment:
As you can see from the CI, this variable is no more needed (or if it is, there is a problem. ;)